plugins/announcements/package.json

```diff
@@ -64,11 +64,11 @@
     "@backstage/plugin-search-common": "^1.2.15",
     "@backstage/plugin-search-react": "^1.8.3",
     "@backstage/plugin-signals-react": "^0.0.7",
+    "@backstage/theme": "^0.5.7",
```

Comment:
is the reverted version but we could check if there is a new version right?

Reply:
Yes, we could, but I thought it would be best to revert 1:1, and then use the backstage-cli to get us to latest version.